fe/fe-core/src/main/java/com/starrocks/common/util/NetUtils.java

```diff
@@ -67,9 +70,25 @@ public static List<InetAddress> getHosts() {
             Enumeration<InetAddress> a = e.getInetAddresses();
             while (a.hasMoreElements()) {
                 InetAddress addr = a.nextElement();
+                // IPv6 address starting with fe80 (Link-local Address) is not supported for now.
+                if (addr instanceof Inet6Address &&
+                        addr.getHostAddress().toLowerCase(Locale.ROOT).startsWith("fe80")) {
```

Comment:
InetAddress has an interface `isLinkLocalAddress()`, looks good for the check?

Reply:
good idea, will fix it